app/services/fact_parser.rb

```diff
@@ -129,6 +129,10 @@ def disks_total
   def kernel_version
   end
 
+  # bios information
+  def bios
```

Comment:
Since `app/models/host_facets/reported_data_facet.rb` assumes it returns a hash, the default implementation should at least return a hash. An empty one is safe enough.
```suggestion
  def bios
    {}
```